src/lib/modules/blockchain_process/proxy.js

```diff
@@ -1,156 +1,178 @@
-const httpProxy = require('http-proxy');
-const http = require('http');
-const constants = require('../../constants.json');
-const utils = require('../../utils/utils');
-
-let commList = {};
-let transactions = {};
-let receipts = {};
+/* global __ exports require */
 
+const Asm = require('stream-json/Assembler');
 const {canonicalHost, defaultHost} = require('../../utils/host');
+const {chain}  = require('stream-chain');
+const cloneable = require('cloneable-readable');
+const constants = require('../../constants.json');
+const express = require('express');
+const {parser} = require('stream-json');
+const proxyMiddleware = require('http-proxy-middleware');
+const pump = require('pump');
+const utils = require('../../utils/utils');
+const WebSocket = require('ws');
+const WsParser = require('simples/lib/parsers/ws');
 
-const parseRequest = function (reqBody) {
-  let jsonO;
+const parseJsonMaybe = (v) => {
+  let o;
```

Comment:
Could you name `v` and `o` to something meaningfull?

Reply:
Will do.

Reply:
Done.